styles.css

```diff
@@ -0,0 +1,911 @@
+  grid-area: top;
+  background-image: linear-gradient(
+      180deg,
+      rgba(236, 143, 27, 0) 0%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(assets/gadget.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50%;
+}
+
+.small-bottom {
+  justify-content: flex-end;
+  grid-area: bottom;
+  background-image: linear-gradient(
+      180deg,
+      rgba(236, 143, 27, 0) 0%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(assets/ev-phantom-rols-royce.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50%;
+}
+
+.highlights-heading-container {
+  height: max-content;
+}
+
+.highlights-heading-container p {
+  color: hsla(0, 0%, 100%, 0.6);
+}
+
+.highlights-heading-container > a {
+  padding-bottom: 20px;
+}
+
+.highlights-heading-container p i {
+  padding: 0 5px 0 5px;
+}
+
+.small-highlight-heading a {
+  font-size: 26px;
+}
+
+@media screen and (max-width: 1000px) {
+  .big-left {
+    background-image: linear-gradient(
+        180deg,
+        rgba(236, 143, 27, 0) 0%,
+        rgba(0, 0, 0, 1) 100%
+      ),
+      url("assets/x-ray.webp");
+  }
+
+  .big-left img {
+    visibility: hidden;
+  }
+}
+
+/**************** Latest News *************/
+.latest {
+  width: 70%;
+  margin: auto;
+}
+
+.latest h2,
+.deals-header {
+  font: 700 25px "avalon-bold", sans-serif;
+  margin-bottom: 20px;
+}
+
+.link-latest {
+  color: #f42;
+  text-decoration: none;
+}
+
+.link-latest::after,
+.funding-section-title::after {
+  display: inline-block;
+  content: "";
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-image: url("assets/arrow.svg");
+  background-size: 16px 16px;
+}
+
+.news-wrapper,
+.deals-wrapper {
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1 1 100%;
+  list-style-type: none;
+}
+
+.news {
+  flex-basis: 25%;
+}
+
+.wrap-news {
+  max-width: 270px;
+  padding: 5px;
+}
+
+.tile {
+  width: 100%;
+  height: 135px;
+}
+
+h4 a {
+  font: 700 16px/1.5 "ars-maquette", sans-serif;
+  color: #000;
+  text-decoration: none;
+}
+
+h4 a:hover {
+  color: #f42;
+  text-decoration: underline;
+}
+
+.story p,
+.sale {
+  font: 12px/18px "ars-maquette", sans-serif;
+  color: #aaa;
+}
+
+@media screen and (max-width: 750px) {
+  #highlights {
+    display: grid;
+    height: 1000px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-areas:
+      "big-left"
+      "top"
+      "bottom";
+  }
+
+  .highlights-item {
+    height: 285px;
+  }
+
+  .highlights-heading-container {
+    height: max-content;
+  }
+
+  .highlights-heading-container a {
+    font-size: 4vw;
+    font-weight: 700;
+    margin-bottom: 1%;
+  }
+
+  .latest {
+    width: 99%;
+  }
+
+  .news-wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .wrap-news {
+    display: flex;
+    max-width: 100%;
+    min-height: 75px;
+  }
+
+  .tile {
+    width: 75px;
+    height: 75px;
+  }
+
+  .story {
+    display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
We fixed all of the requested changes per first code review. Requesting a new code review